### ISSUE TITLE
Issue when executing the script

### ISSUE THREAD

**Issue by thebest8815** [Frankenmint/PKGenerator_Checker#10]
When I execute the code, it shows the following:

C:\Users\Mike\Documents\Fun\PKGenerator_Checker>python PkMaker.py
Traceback (most recent call last):
  File "PkMaker.py", line 65, in <module>
    pk = os.urandom(32).encode("hex")
AttributeError: 'bytes' object has no attribute 'encode'

Any idea?

**Comment by Frankenmint** [comment]
yeah this was happening because you were attempting to run this using python3.  I've decided to dust off the ole script and update it to python 3 as well.  Took more time than I expected, but it was fun.  Hope you like it!
